### DISCUSSION TITLE
What is ADE

### DISCUSSION THREAD

**Discussion by gadiLhv** [thliebig/openEMS-Project#91] [General]
Hello there

A bit of an unrelated question.

I am trying to build an engine extension, and basing it on the Lorentz material engine extension (that derives from the dispersive material one). 

I was wondering what is meaning of the acronym **ADE**?

`//! ADE currents
	// Array setup: curr_ADE[N_order][direction][mesh_pos]
	FDTD_FLOAT ***curr_ADE;`

`//! ADE voltages
	// Array setup: volt_ADE[N_order][direction][mesh_pos]
	FDTD_FLOAT ***volt_ADE;`

**Comment by biergaizi** [comment]
Auxiliary Differential Equation. I'm not a physicist and I don't pretent I can understand them, but just search the keyword "ADE-FDTD", and you can find lots of literature on its use in Lorentz metal modeling for FDTD.
